pkg/kubectl/cmd/annotate.go

```diff
@@ -92,6 +93,7 @@ var (
 )
 
 func NewCmdAnnotate(f cmdutil.Factory, out io.Writer) *cobra.Command {
+	LoadTranslations()
```

Comment:
Actually, I take that back.  Since the eventual impl is probably a properties file compiled into the binaries for most code paths init is much cheaper.  Although I wouldn't want to decorate all the command code paths with init code.  Maybe lazy is better there, or just an init early in code.

I don't think single bundle for binary is going to work for hyperkube, so we're going to need packages per bundle type.